lhotse/cut/multi.py

```diff
@@ -84,172 +73,10 @@ class MultiCut(Cut):
-
-    def load_custom(self, name: str) -> np.ndarray:
-        """
-        Load custom data as numpy array. The custom data is expected to have
-        been stored in cuts ``custom`` field as an :class:`~lhotse.array.Array` or
-        :class:`~lhotse.array.TemporalArray` manifest.
-
-        .. note:: It works with Array manifests stored via attribute assignments,
-            e.g.: ``cut.my_custom_data = Array(...)``.
-
-        :param name: name of the custom attribute.
-        :return: a numpy array with the data.
-        """
-        from lhotse.array import Array, TemporalArray
-
-        value = self.custom.get(name)
-        if isinstance(value, Array):
-            # Array does not support slicing.
-            return value.load()
-        elif isinstance(value, TemporalArray):
-            # TemporalArray supports slicing.
-            return value.load(start=self.start, duration=self.duration)
-        elif isinstance(value, Recording):
-            # Recording supports slicing.
-            return value.load_audio(
-                channels=self.channel, offset=self.start, duration=self.duration
-            )
-        else:
-            raise ValueError(
-                f"To load {name}, the cut needs to have field {name} (or cut.custom['{name}']) "
-                f"defined, and its value has to be a manifest of type Array or TemporalArray."
-            )
-
-    @property
-    def recording_id(self) -> str:
-        return self.recording.id if self.has_recording else self.features.recording_id
-
-    @property
-    def has_features(self) -> bool:
-        return self.features is not None
-
-    @property
-    def has_recording(self) -> bool:
-        return self.recording is not None
-
-    def has(self, field: str) -> bool:
-        if field == "recording":
-            return self.has_recording
-        elif field == "features":
-            return self.has_features
-        else:
-            return self.custom is not None and field in self.custom
-
-    @property
-    def frame_shift(self) -> Optional[Seconds]:
-        return self.features.frame_shift if self.has_features else None
-
-    @property
-    def num_frames(self) -> Optional[int]:
-        return (
-            compute_num_frames(
-                duration=self.duration,
-                frame_shift=self.frame_shift,
-                sampling_rate=self.sampling_rate,
-            )
-            if self.has_features
-            else None
-        )
-
-    @property
-    def num_samples(self) -> Optional[int]:
-        return (
-            compute_num_samples(self.duration, self.sampling_rate)
-            if self.has_recording
-            else None
-        )
-
-    @property
-    def num_features(self) -> Optional[int]:
-        return self.features.num_features if self.has_features else None
-
     @property
     def num_channels(self) -> Optional[int]:
         return len(self.channel) if self.has_recording else None
```

Comment:
This should always be `len(self.channel)`, similarly like it's always `1` in MonoCut. Even if you drop the recording, the precomputed features represent these channels.